components/python/psycopg2/psycopg2-PYVER.p5m

```diff
@@ -9,10 +9,10 @@
 
 #
 # Copyright 2016 Alexander Pyhalov
-# Copyright 2021 Andreas Wacknitz
+# Copyright 2021, 2022 Andreas Wacknitz
 #
 
-set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
Here is the direct result of the confusion: the file is named `...-PYVER.p5m`, but the fmri lacks `-$(PYV)`.